### ISSUE TITLE
Magento2.Commenting.ClassPropertyPHPDocFormatting fails to detect comment for typed class properties

### ISSUE THREAD

**Issue by barbazul** [squizlabs/PHP_CodeSniffer#3439]
**Describe the bug**

When using typed class properties the "Missing PHP DocBlock for class property." warning is triggered

**Code sample**
```php
<?php

namespace Vendor\Extension\Controller\Adminhtml\Stuff;

use Magento\Backend\App\Action;
use Magento\Sales\Api\Data\OrderInterface;

class DoStuff extends Action {
    /**
     * @var OrderRepositoryInterface
     */
    private OrderRepositoryInterface $orderRepository;

    // More controller code
}
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```
 12 | WARNING | Missing PHP DocBlock for class property.
```

**Expected behavior**
No warning should be raised as the property has a corresponding docblock.

**Versions (please complete the following information):**
 - OS: Linux Mint 20.2 Cinnamon
 - PHP: 7.4.23
 - PHPCS: 3.6.0
 - Standard: Magento2

**Additional context**
If we remove the type definition, the warning dissapears.

**Comment by barbazul** [comment]
Oops. Wrong repo.
